threadbare/operations.py

```diff
@@ -265,6 +274,13 @@ def _print_line(output_pipe, line, **kwargs):
 
         # render template and write to given pipe
         template = final_kwargs["line_template"]
+
+        if not final_kwargs["display_prefix"]:
+            try:
+                template = template[template.index("{line}") :]
+            except ValueError:  # "substring not found"
+                pass
```

Comment:
add a LOG.warn here, I think the dev should know their `display_prefix` isn't working.